components/gitpod-cli/cmd/timeout-show.go

```diff
@@ -41,7 +41,7 @@ var showTimeoutCommand = &cobra.Command{
 		if err != nil {
 			fail(err.Error())
 		}
-		fmt.Printf("Workspace timeout is set to %d minutes.\n", int(duration.Minutes()))
+		fmt.Printf("Current workspace timeout: %d minutes.\n", int(duration.Minutes()))
```

Comment:
**Suggestion:** I don't think we have a strong enough reason to change this, also it's somewhat of an "API" if there are users relying on this output for whatever reason, can we omit the change from the PR?